### DISCUSSION TITLE
[Solved: wrong project, moved to OpenDTU-onBattery] SML Powermeter funktioniert seit Vers. 2024.3.15 nicht mehr

### DISCUSSION THREAD

**Discussion by drucknase** [tbnobody/OpenDTU#2013] [Q&A]
opendtu-onbattery-generic_esp32.bin:
Ich habe seit der Version 2024.03.07 kein Update der Firmware gemacht und vor wenigen Tagen ein Update auf Ver. 2024.05.07 durchgeführt. Seit dem funktioniert das Auslesen des SML (OBIS) Powermeter nicht mehr.
Ich habe alle Versionen zwischen 2024.05.07 und 2024.03.07 ausprobiert und bei keiner Version hat die SML Schnittstelle zum Powermeter funktioniert. Mir ist auch aufgefallen, dass die Filegröße der Vers. 2024.3.15, 2024.3.22 nur 1,57MB betragen, ab Vers. 2024.03.23 die Filegröße wieder 1,86MB betrug.
Nachdem was ich bis jetzt in den Source Files recherchiert habe, fehlt die Definition bzw. Konfiguration für den POWERMETER_PIN_RX 
Wo muss dieser Wert für den seriellen RX konfiguriert werden?
Danke für eure Unterstützung.

**Comment by MetaChuh** [comment]
@drucknase 

danke dir für den onbattery solved tag hier.
zusatzbitte: wenn du zeit hast, schreibe (copy-paste) zusätzlich hier die lösung zu deinem issue und markiere diese als solution.

grund: wenn unsere besucher "solved" lesen, nachdem sie diese discussion anhand ihrer eigenen anforderungs und suchbegriffen gefunden haben, sollte exakt dafür auch eine explizite lösung hier verfügbar sein.

thx & greetings,
metachuh
